frame/support/procedural/src/storage/print_pallet_upgrade.rs

```diff
@@ -323,8 +323,8 @@ pub mod pallet {{
 	#[pallet::generate_store({store_vis} trait Store)]
 	pub struct Pallet{decl_gen}(PhantomData{use_gen_tuple});
 
-	#[pallet::interface]
-	impl{impl_gen} Hooks<BlockNumberFor<T>> for Pallet{use_gen}
+	#[pallet::hooks]
+	impl{impl_gen} Hooks for Pallet{use_gen}
```

Comment:
```suggestion
	impl{impl_gen} Hooks<BlockNumberFor<T>> for Pallet{use_gen}
```
thanks for the name fix, but the hooks generic is still required no ?

Reply:
Indeed it is!